platforms/chibios/converters/promicro_to_elite_pi/_pin_defs.h

```diff
@@ -0,0 +1,43 @@
+// Copyright 2022 QMK
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+// Left side (front)
+#define D3 0U
+#define D2 1U
+//      GND
+//      GND
+#define D1 2U
+#define D0 3U
+#define D4 4U
+#define C6 5U
+#define D7 6U
+#define E6 7U
+#define B4 8U
+#define B5 9U
+
+// Extra Elite-C compatible pins
+#define B7 12U
+#define D5 13U
+#define C7 14U
+#define F1 15U
+#define F0 16U
```

Comment:
Extra pins that are not part of a promicro shouldnt really be defined here
```suggestion
```

Reply:
Converters that define bottom row? Those would be incorrect for a "pro micro" converter.

However it could be valid for a `elite_c` converter, be it that there is currently the assumption for a single compatible pin type.

Reply:
I'm okay without the extra pins, basically was just copying the style of the other converters, but now I see in the new PR that we'll axe them.